hoomd/md/methods.py

```diff
@@ -19,8 +19,37 @@
 import copy
 from collections.abc import Sequence
 
+
 class _Method(_HOOMDBaseObject):
-    pass
+    """Base class integration method.
+
+    Provides common methods for all subclasses.
+
+    Note:
+        Users should use the subclasses and not instantiate `_Method` directly.
+    """
+
+    def thermalize_extra_dof(self, seed):
+        """Set the extra degrees of freedom to random values.
+
+        Args:
+            seed (int): Random number seed
+
+        Some integration methods, such as `NVT` and `NPT` employ extra degrees
+        of freedom for the thermostat and/or barostat. `thermalize_extra_dof`
+        sets these to random values. This method performs no operation for
+        integration methods that lack internal degrees of freedom.
```

Comment:
The reason for the above comment is expressed in these lines. I worry that it may be confusing to users to have this function as a no-op for most integrators although most common use cases would indeed have this perform a meaningful action.

Reply:
We can implement in only those methods where it does something: NVT, NPT, and NPH.